WalletWasabi/WabiSabi/Client/CoinJoin/Client/CoinJoinClient.cs

```diff
@@ -126,6 +126,17 @@ private async Task<RoundState> WaitForBlameRoundAsync(uint256 blameRoundId, Canc
 			throw new InvalidOperationException($"Blame Round ({roundState.Id}): Abandoning: the round is not economic.");
 		}
 
+		if (roundState.CoinjoinState.Parameters.CoordinationFeeRate.Rate * 100 > CoinJoinConfiguration.MaxCoordinationFeeRate)
```

Comment:
Simply comment: the `MaxCoordinationFeeRate` is not a rate but a percentage. We have to fix these things one day because they are confusing and lead to errors.

Reply:
I fixed it internally